src/averageAges.js

```diff
@@ -14,12 +14,25 @@
  *
  * @return {number}
  */
+function getFilteredGender(people, gender) {
+  return people.filter(genderCheck => genderCheck.sex === gender);
+}
+
+function getAverageAge(people) {
+  const TotalAge = people.map(person => person.died - person.born);
+  const AverageAge = TotalAge.reduce((total, current) =>
+    total + current) / TotalAge.length;
+
+  return AverageAge;
+};
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = !century
+    ? getFilteredGender(people, 'm')
+    : getFilteredGender(people, 'm').filter(man => (
+      century === Math.ceil(man.died / 100)));
```

Comment:
better store your `getFilteredGender` result in a variable and then filter it if needed, don't repeat yourself